managers/paymaster-operations-manager.js

```diff
@@ -0,0 +1,101 @@
+export default class PaymasterOperationsManager {
+    constructor(services) {
```

Comment:
changes look better now, but isn't the main point of the paymaster that you can use this contract instead of your wallet to publish? so we need some logic in create, and other functions that spend TRAC to use paymaster instead of your own wallet

Reply:
probably good idea to have that as part of this pr, otherwise feature is not useful i think?